### DISCUSSION TITLE
date of payment

### DISCUSSION THREAD

**Discussion by df911** [firefly-iii/firefly-iii#5124] [General]
Hello,

When the date of payment in credit card section is used ? 
I dont' understand the features and there is no information in documentation.

Thanks

**Comment by JC5** [comment]
I don't understand the question, please elaborate.

**Comment by df911** [comment]
I don't understand how to use this feature : 
![carpture](https://user-images.githubusercontent.com/64459680/134876006-8cef41ba-446b-4a64-b789-7e1dfaaff97d.png)

**Reply by JC5** [reply]
It's not used at the moment, just like it states below the date input.
